packages/core/src/debug.test.ts

```diff
@@ -11,131 +34,159 @@ test("core IR type layouts", () => {
   });
 });
 
-test("test rose to rust formatting", () => {
-  const f = fn([Real, Real], Real, (x, y) => add(mul(x, 2), y));
-  const g = wasm.js2Rust(f.f.f);
-  // TODO: maybe make a more readable IR text format
-  expect(g).toBe(
-    `Function {
-    generics: [],
-    types: [
-        Bool,
-        F64,
-        Tuple {
-            members: [
-                Ty(
-                    1,
-                ),
-                Ty(
-                    1,
-                ),
-            ],
-        },
-    ],
-    funcs: [],
-    param: Ty(
-        2,
-    ),
-    ret: Ty(
-        1,
-    ),
-    vars: [
-        Ty(
-            2,
-        ),
-        Ty(
-            1,
-        ),
-        Ty(
-            1,
-        ),
-        Ty(
-            1,
-        ),
-        Ty(
-            1,
-        ),
-        Ty(
-            1,
-        ),
-    ],
-    blocks: [
-        Block {
-            arg: Var(
-                0,
-            ),
-            code: [
-                Instr {
-                    var: Var(
-                        1,
-                    ),
-                    expr: Member {
-                        tuple: Var(
-                            0,
-                        ),
-                        member: Member(
-                            0,
-                        ),
-                    },
-                },
-                Instr {
-                    var: Var(
-                        2,
-                    ),
-                    expr: Member {
-                        tuple: Var(
-                            0,
-                        ),
-                        member: Member(
-                            1,
-                        ),
-                    },
-                },
-                Instr {
-                    var: Var(
-                        3,
-                    ),
-                    expr: F64 {
-                        val: 2.0,
-                    },
-                },
-                Instr {
-                    var: Var(
-                        4,
-                    ),
-                    expr: Binary {
-                        op: Mul,
-                        left: Var(
-                            1,
-                        ),
-                        right: Var(
-                            3,
-                        ),
-                    },
-                },
-                Instr {
-                    var: Var(
-                        5,
-                    ),
-                    expr: Binary {
-                        op: Add,
-                        left: Var(
-                            4,
-                        ),
-                        right: Var(
-                            2,
-                        ),
-                    },
-                },
-            ],
-            ret: Var(
-                5,
-            ),
-        },
-    ],
-    main: Block(
-        0,
-    ),
-}`,
+test("pprint if", () => {
```

Comment:
Minor stylistic change: could you put all the `pprint` tests under a single `describe`? E.g.:

```typescript
describe("pprint", () => {
  test("if", () => { /* ... */ });
  test("call funcs", () => { /* ... */ });
  test("unary operations", () => { /* ... */ });
  test("binary operations", () => { /* ... */ });
});
```